### PR TITLE
Video Service

### PR DESCRIPTION
**Pull request by Elad-Laufer**

### CODE REVIEW

File: media_platform/job/extract_poster_job.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from media_platform.service.rest_result import RestResult
+from media_platform.service.source import Source
+
+
+class ExtractPosterSpecification(Specification):
+    def __init__(self, second, destination, image_format='jpg'):
+        # type: (float, Destination, str) -> None
+        super(ExtractPosterSpecification, self).__init__()
+
+        self.second = second
```
Comment (golanpa) [inline media_platform/job/extract_poster_job.py]: what is **second** ? is it, _from_second_ ?

File: media_platform/job/extract_poster_job.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        self.destination = destination
+        self.format = image_format or 'jpg'
```
Comment (golanpa) [inline media_platform/job/extract_poster_job.py]: verify that we support input image_format?

File: media_platform/job/extract_storyboard_job.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+
+class ExtractStoryboardSpecification(Specification):
+    def __init__(self, destination, columns, rows, tile_width=None, tile_height=None, image_format='jpg',
+                 segment_duration=None):
+        # type: (Destination, int, int, int, int, str, float) -> None
+        super(ExtractStoryboardSpecification, self).__init__()
+        self._validate_image_format(image_format)
+
+        if image_format == 'jpg' and tile_width:
```
Comment (golanpa) [inline media_platform/job/extract_storyboard_job.py]: the two ifs should be extracted into a method

File: media_platform/job/extract_poster_job_group.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from media_platform.job.job_group import JobGroup
+
+
+class ExtractPosterJobGroup(JobGroup):
+
+    @classmethod
+    def deserialize(cls, data):
```
Comment (golanpa) [inline media_platform/job/extract_poster_job_group.py]: is tI guess it is done to help IDE to figure out types

File: media_platform/job/extract_poster_job.py (hunk 1 above)
Comment (Elad-Laufer) [reply media_platform/job/extract_poster_job.py]: the point in time from which the poster is taken

File: media_platform/job/extract_poster_job_group.py (hunk 4 above)
Comment (Elad-Laufer) [reply media_platform/job/extract_poster_job_group.py]: yes

File: media_platform/job/extract_storyboard_job.py (hunk 3 above)
Comment (Elad-Laufer) [reply media_platform/job/extract_storyboard_job.py]: fair enough

File: media_platform/job/extract_poster_job.py (hunk 2 above)
Comment (Elad-Laufer) [reply media_platform/job/extract_poster_job.py]: ok

File: media_platform/job/extract_poster_job.py (hunk 1 above)
Comment (golanpa) [reply media_platform/job/extract_poster_job.py]: I figured it out but i do not think it will be clear to the user of the SDK. I suggested _from_second_

**Pull request merged**